src/locales/en.json

```diff
@@ -1,14 +1,14 @@
 {
   "nav": {
-    "item_files": "Files",
+    "item_files": "Drive",
```

Comment:
Same comment than for the breadcrumb, @pierrevdk will decide what they want, but as for now we copy what Google does (= using Drive everywhere).